pkg/common/config/config.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+import (
+	"io/ioutil"
+	log "github.com/sirupsen/logrus"
+)
+
+var initContainerTemplate string
+
+func init() {
+	bytes, err := ioutil.ReadFile("/etc/config/initContainer.yaml")
```

Comment:
Why read init container template from a file of hard coded path? Is it better to add a new arg to the cmd to accept the init container template file?

Reply:
modified